src/components/DocumentSelector/style.css

```diff
@@ -14,14 +14,14 @@
   width: 100%;
   background: @color-white;
   border-radius: 4*@unit;
-  border: 1px solid @color-selectors-border;
+  border: @selectors-border;
   color: @color-text-dark;
   cursor: pointer;
   display: flex;
   font: inherit;
   line-height: 1.43;
   margin-bottom: 8*@unit;
-  padding: 16*@unit;
+  padding: calc(16*@unit);
```

Comment:
Why do you need `calc` here?

Reply:
Oops. Missed that when I removed a change I made here in an earlier commit.